packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -205,29 +179,35 @@ const ChartMarkers = connect(({ modules, ui, client }) => ({
     currency: client.currency,
 }))(Markers);
 
-class ChartTradeClass extends React.Component {
-    state = {
-        active_category: null,
-    };
-    bottomWidgets = ({ digits, tick }) => <ChartBottomWidgets digits={digits} tick={tick} />;
-    topWidgets = ({ ...props }) => {
-        const { is_digits_widget_active, try_synthetic_indices } = this.props;
+const ChartTradeClass = props => {
```

Comment:
Could we use a better name instead of `ChartTradeClass`?

Reply:
renamed this to `Chart`